src/screens/HyperSwitch/Home/HomeV2.res

```diff
@@ -397,15 +397,21 @@ module QuickStartModule = {
 
 @react.component
 let make = () => {
-  let featureFlagDetails = HyperswitchAtom.featureFlagAtom->Recoil.useRecoilValueFromAtom
-  <div className="flex flex-col gap-5">
-    <UIUtils.RenderIf condition={featureFlagDetails.acceptInvite}>
-      <AcceptInviteHome />
-    </UIUtils.RenderIf>
-    <div className="w-full flex flex-col gap-14">
-      <QuickStartModule />
+  let {isProdIntentCompleted} = React.useContext(GlobalProvider.defaultContext)
+  let enumDetails = Recoil.useRecoilValueFromAtom(HyperswitchAtom.enumVariantAtom)
+  let typedEnumValue = enumDetails->LogicUtils.safeParse->QuickStartUtils.getTypedValueFromDict
+  let showRecipesAndPlugins =
+    [
+      typedEnumValue.testPayment.payment_id->LogicUtils.isNonEmptyString,
+      typedEnumValue.integrationCompleted,
+      isProdIntentCompleted->Option.getOr(false),
+    ]->Array.includes(false)
```

Comment:
there will be UI if you directly check for these values.Please refer to QuickStartModule

Reply:
updated